andela/boromir-cfh
# #146622697  save game logs of every game

### What does this PR do?
It allows users to create/start a new game

Description of Task to be completed?
-Users should be able to create/start a new game
-after every game, the game details is saved in the database

How should this be manually tested?
- start and complete a game session

Any background context you want to provide?
-create a table ```gamelogs``` in your database
-i have created endpoint  GET ```/api/games/save``` with method ```getLogs``` in the ```Gamelog``` controller for the person working on  task ```create a leaderboard```

What are the relevant pivotal tracker stories?
-146622697

Users should be able to start a new game session after login or signup.
Screenshots (if appropriate)
